.github/workflows/deploy.yml

```diff
@@ -28,12 +29,23 @@ jobs:
       - name: Install deps
         run: pnpm i
 
-      - name: Build the code
-        run: pnpm run build
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Publish to NPM & Docker
+        run: pnpm publish
```

Comment:
I don't see where we override this functionality to publish to docker as well, AFAIK this only publishes to NPM...

Reply:
Oversight. My bad.